docs/reference/rest-api/security/grant-api-keys.asciidoc

```diff
@@ -83,15 +86,28 @@ It supports nested data structure.
 Within the `metadata` object, keys beginning with `_` are reserved for
 system usage.
 
+`client_authentication`::
+(Optional, object) When using the `access_token` grant type, and when supplying a
+JWT, this specifies the client authentication for <<JWT access tokens, jwt-realm-oauth2>>
+ (i.e. what's normally specified by the `ES-Client-Authentication` request header).
+
+`scheme`:::
+(Required, string) The scheme (case-sensitive) as it's supplied in the
+`ES-Client-Authentication` request header.
```

Comment:
Can we add reference to 'SharedSecret' since that is the only supported value.

Reply:
I've pushed [86c7991](https://github.com/elastic/elasticsearch/pull/101904/commits/86c79915c5ea4013db7b0adad00d7f564c9594ea) to reference it from the JWT authentication page.